rust-runtime/aws-smithy-client/src/lib.rs

```diff
@@ -196,9 +209,13 @@ where
             Service<Operation<O, Retry>, Response = SdkSuccess<T>, Error = SdkError<E>> + Clone,
     {
         let connector = self.connector.clone();
+
         let svc = ServiceBuilder::new()
-            // Create a new request-scoped policy
+            .layer(TimeoutLayer::new(self.timeout_config.api_call_timeout()))
```

Comment:
might want to workshop api_call_timeout vs. api_call_attempt_timeout. Could definitely see people getting confused between them

Reply:
I agree. I think the distinction we want to communicate with these names is that one timeout is for a single HTTP request and one timeout is for a "logical" request to an API consisting of 1+ HTTP requests.

API call timeout -> API Request Timeout
API call attempt timeout -> HTTP Request Timeout